modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/display/context/CommercePriceListDisplayContext.java

```diff
@@ -260,20 +260,42 @@ public long getParentCommercePriceListId() throws PortalException {
 		return commercePriceList.getParentCommercePriceListId();
 	}
 
-	public CreationMenu getPriceListCreationMenu() throws Exception {
+	public CreationMenu getPriceListCreationMenu(String portletName)
+		throws Exception {
+
 		CreationMenu creationMenu = new CreationMenu();
 
+		String title = StringPool.BLANK;
```

Comment:
Hey @elocamp , what's the purpose of this variable?

Reply:
Oops, i forgot to delete this one, so sorry

Reply:
Move it after the permissions check and reassign it based on `portletName`